* use AM_GNU_GETTEXT macro, but do not enable, it requires gettext to be
  installed in order to run autoconf
* make sure we really don't need gettext

*--- configure.ac
*+++ configure.ac
*@@ -241,15 +241,16 @@
* dnl -------
* 
* 
*-dnl AM_GNU_GETTEXT
*+#NLSENABLE AM_GNU_GETTEXT([external])
*+#NLSENABLE HAVEGETTEXT="${USE_NLS}"
*+#NLSENABLE INTLIBS="${LIBINTL}"
* 
* 
*-
*-INTLIBS=""
*-
*-AC_CHECK_LIB(intl, gettext, [INTLIBS="-lintl" [HAVEGETTEXT="yes"]], [
*-	INTLIBS=""
*-	AC_CHECK_FUNC(gettext, [HAVEGETTEXT="yes"]) ])
*+dnl INTLIBS=""
*+dnl 
*+dnl AC_CHECK_LIB(intl, gettext, [INTLIBS="-lintl" [HAVEGETTEXT="yes"]], [
*+dnl 	INTLIBS=""
*+dnl 	AC_CHECK_FUNC(gettext, [HAVEGETTEXT="yes"]) ])
* 
* AC_CHECK_PROG(XGETTEXT, xgettext, xgettext)
* 
--- src/misc.c
+++ src/misc.c
@@ -645,13 +645,13 @@
 
     if(advanced)
         rv = wAdvancedInputDialog(scr,
-		title ? gettext(title):_("Program Arguments"),
-		prompt ? gettext(prompt):_("Enter command arguments:"),
+		title ? _(title):_("Program Arguments"),
+		prompt ? _(prompt):_("Enter command arguments:"),
 		name, &ret);
     else
         rv = wInputDialog(scr,
-		title ? gettext(title):_("Program Arguments"),
-		prompt ? gettext(prompt):_("Enter command arguments:"),
+		title ? _(title):_("Program Arguments"),
+		prompt ? _(prompt):_("Enter command arguments:"),
 		&ret);
 
     if(title) wfree(title);
--- src/rootmenu.c
+++ src/rootmenu.c
@@ -363,8 +363,11 @@
     if(!strncmp(NOTRANSLATE_PREFIX, title, strlen(NOTRANSLATE_PREFIX)))
 	return title + strlen(NOTRANSLATE_PREFIX);
     if(wPreferences.translate_menu)
+	#if defined(HAVE_LIBINTL_H) && defined(I18N)
 	return dgettext("menu-messages", title);
+	#else
     return title;
+	#endif
 }
 
 /********************************************************************/
